### PR TITLE
[bitnami/logstash] Add tests and publishing using VIB

### PR DESCRIPTION
**Pull request by FraPazGal**
Signed-off-by: FraPazGal <fdepaz@vmware.com>

### Description of the change

The main objective of this PR is to publish our Bitnami MongoDB Sharded Helm Chart using VMware Image Builder. In order to do that, several changes are included:

- Adding the asset to the publishing workflow
- Increasing the existing test coverage of the asset by adding GOSS tests.

### Benefits

- Ensuring higher quality of the Helm charts.
- Increased pool of assets completely handled by VMware Image Builder.

### Possible drawbacks

Automated tests could show to be potentially flaky. 

### Applicable issues

NA

### Additional information

Tested in https://github.com/FraPazGal/charts/pull/109

### Checklist

- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: .vib/logstash/goss/goss.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    - /port.*{{ .Vars.containerPorts.http }}/
+  /var/run/secrets/kubernetes.io/serviceaccount:
+    exists: {{ .Vars.serviceAccount.automountServiceAccountToken }}
+    filetype: directory
+    mode: "3777"
+command:
+  {{- $rnd_address := printf "http://%s.com" (randAlpha 5) }}
+  test-pipeline:
+    # Logstash can take some time to parse and filter each input
+    exec: echo '98.83.179.51 - - [18/May/2011:19:35:08 -0700] "GET /css/main.css HTTP/1.1" 200 55 "{{ $rnd_address }}" "Mozilla/5.0"' >> /tmp/test_input && sleep 20 && cat /tmp/test_log
```
Comment (agomezmoron) [inline .vib/logstash/goss/goss.yaml:25]: If the IP `98.83.179.51` is "only to have one", I'd use another random one. I think is smarter.

**Review by agomezmoron — APPROVED** [review]
Thanks for the contribution! I left a minor comment like "nice to have". LGTM

**Review by joancafom — DISMISSED** [review]
LGTM, with agomezmoron's feedback

**Review by agomezmoron — APPROVED** [review]
Thank you for the changes! LGTM

**Review by joancafom — APPROVED** [review]

**Pull request merged**